### ISSUE TITLE
BMW Call button

### ISSUE THREAD

**Issue by toadpole** [nerfherder/BluetoothGoogleNow#1]
Doesn't work with the call button on my BMW car kit steering controls. This button usually redials. After losing the module, it just doesn't do anything.

**Comment by toadpole** [comment]
Sorry after *installing the module

**Comment by nerfherder** [comment]
Hi toadpole, does the redial button launch google now if the phone is on and unlocked? Just checking to see if this is the same as Issue #2.

**Comment by toadpole** [comment]
Nope, screen status doesn't matter, doesn't do anything either way. One weird thing is, it does respond to the click, but instead of re dialling as it always did, it brings up the phone menu on the screen when the module is activated. When I disable the module, it goes back to redial

**Comment by nerfherder** [comment]
I have released a new version which uses a different intent to call up google now, can you see if this works now?

**Comment by toadpole** [comment]
Nope, same thing, just brings up phone menu on screen instead of the original redial function.

**Comment by djpole** [comment]
Same problem here with my BMW 530 e60. Redial button on steering wheel always launches original redial function both on phone's screen and car's screen. Screen status doesn't matter, too.
I think the BT module in these car's is very problematic and there's no solution. I have tried different Tasker profiles to call Google now , and the microphone and speakers in the car don't react to my voice but the phone's ones do. SCO profile works someties, but only for play and repeat, not for listening.
Very difficult solution for us.
